services/system.spec.js

```diff
@@ -0,0 +1,55 @@
+import { getSystemData } from '@/services'
+
+describe('getSystemData function', () => {
+  let systemData
+
+  beforeEach(async () => {
+    systemData = await getSystemData('1')
+  })
```

Comment:
Pensando em "otimização", aqui poderia ser um `beforeAll`, porque você não tá alterando nada do `systemData` nos testes, certo?

Reply:
@jooaodanieel agora acho que ficou bom mesmo! Pensei que tinha que colocar os testes dentro da pasta tests, mas o jest já busca por arquivos `*.spec.js` em todos os diretórios do projeto. Também não lembrava que tinha o método `it` no jest, eu já estava pensando em fazer os testes no estilo que nós fazemos no solus lá com o rails (estilo `it 'returns ...'`). E quebrar os expects em mais testes realmente melhorou muito a legibilidade.
Fiz mais um commit alterando o `beforeEach` para `beforeAll`, só para otimizar.